tests/std/tests/VSO_0180469_ptr_cat/test.cpp

```diff
@@ -214,7 +214,7 @@ void ptr_cat_test_cases() {
 
 template <bool Expected, class Ptr1, class Ptr2, class Pr>
 void test_case_Equal_memcmp_is_safe_helper() {
-    STATIC_ASSERT(decltype(_Equal_memcmp_is_safe(declval<Ptr1>(), declval<Ptr2>(), declval<Pr>()))::value == Expected);
+    STATIC_ASSERT(_Equal_memcmp_is_safe<Ptr1, Ptr2, Pr> == Expected);
 }
 
 template <bool Expected, class Elem1, class Elem2>

```

Comment:
As mentioned in the other comment, if we make the variable template ignore top-level `const` on pointers, we should add test cases to `test_case_Equal_memcmp_is_safe()` below.

Reply:
Added some more tests